Lurk/Evaluation/Expr.lean

```diff
@@ -146,39 +146,39 @@ where
     | _ => (expr, bindAcc)
 
 open Std Format in
-partial def toFormat (e : Expr) : Format := 
+partial def toFormat (esc := false) (e : Expr) : Format := 
   let _ : ToFormat Expr := ⟨toFormat⟩ 
   match e with
   | .lit l => format l
   | .sym s => format s
```

Comment:
This is where `esc` should be used, right?

Reply:
Yes